tests/unit/filing-template-mixin.spec.ts

```diff
@@ -72,32 +75,34 @@ describe('Registration Filing', () => {
         streetAddressAdditional: 'Suite 2'
       }
     }
-    store.state.stateModel.registration.naics = {
+    store.stateModel.registration.naics = {
       naicsCode: '12345',
       naicsDescription: 'Some NAICS Description'
     }
-    store.state.stateModel.nameRequest = {
-      legalType: 'SP',
+    store.stateModel.nameRequest = {
+      legalType: CorpTypeCd.SOLE_PROP,
       nrNum: 'NR 1234567'
     }
-    store.state.stateModel.nameRequestApprovedName = 'My Approved Name'
-    store.state.stateModel.businessContact = {
+    store.stateModel.nameRequestApprovedName = 'My Approved Name'
+
+    const extension: any = '444'
```

Comment:
Looks like this is just used on line 92?

If so then I'd hard-code it like it was before.

Reply:
I will revert it the way it was. I was trying to fix a TS error but  I can take a look at it later :)

Reply:
Probably a type mismatch (number vs string).

Reply:
Yeah, JS is pretty loose that way, but TS tries to get us to use the types correctly 😀